### ISSUE TITLE
Example code won't run with Python 2.7.12

### ISSUE THREAD

**Issue by hdon** [neogeny/TatSu#47]
I ran `pip install tatsu` in my `virtualenv` and tried evaluating the example arithmetic parser as found at https://pypi.python.org/pypi/tatsu (the exact code I ran is included below) and the exact output with uncaught exception can be found below:

```
$ python testtatsu.py
# PPRINT
[ u'3',
  u'+',
  [ u'5',
    u'*',
    [ u'(',
      [ u'10',
        u'-',
        u'20'],
      u')']]]
()
# JSON
Traceback (most recent call last):
  File "testtatsu.py", line 46, in <module>
    print(json.dumps(ast.asjson(), indent=2))
AttributeError: 'list' object has no attribute 'asjson'
```

I'm running Python 2.7.12 on Ubuntu 16.04. Here's the output of my `pip freeze` and apologies for some of the packages listed not being relevant:

```
asn1crypto==0.22.0
bcrypt==3.1.3
cffi==1.11.0
cryptography==2.0.3
enum34==1.1.6
idna==2.6
ipaddress==1.0.18
paramiko==2.3.0
pkg-resources==0.0.0
pyasn1==0.3.5
pycparser==2.18
PyNaCl==1.1.2
six==1.11.0
TatSu==4.2.3
```

Here's the arithmetic example I ran:

```python
GRAMMAR = '''
    @@grammar::CALC


    start = expression $ ;


    expression
        =
        | expression '+' term
        | expression '-' term
        | term
        ;


    term
        =
        | term '*' factor
        | term '/' factor
        | factor
        ;


    factor
        =
        | '(' expression ')'
        | number
        ;


    number = /\d+/ ;
'''


if __name__ == '__main__':
    import pprint
    import json
    from tatsu import parse

    ast = parse(GRAMMAR, '3 + 5 * ( 10 - 20 )')
    print('# PPRINT')
    pprint.pprint(ast, indent=2, width=20)
    print()

    print('# JSON')
    print(json.dumps(ast.asjson(), indent=2))
    print()
```

**Comment by apalala** [comment]
@hdon Thanks! Your report is correct. The problem is solved in the master version of the repository, pending a release.

**Comment by apalala** [comment]
Fixed in 4.2.4
